components/Buttons/src/MDCRaisedButton.m

```diff
@@ -26,6 +25,7 @@ + (void)initialize {
                                     forState:UIControlStateNormal];
   [[MDCRaisedButton appearance] setElevation:MDCShadowElevationRaisedButtonPressed
                                     forState:UIControlStateHighlighted];
+  [[MDCRaisedButton appearance] setContentEdgeInsets:MDCButtonDefaultContentEdgeInsets];
```

Comment:
Isn't this already set in +[MDCButton initialize] ?

Reply:
UIAppearance calls applied to MDCButton don't apply to subclasses of MDCButton.

Reply:
Of course. ack.